desktop/src/main/java/bisq/desktop/util/FormBuilder.java

```diff
@@ -1519,6 +1519,15 @@ private static Label getTopLabel(String title) {
         return addTopLabelTextFieldWithCopyIcon(gridPane, rowIndex, colIndex, title, value, -Layout.FLOATING_LABEL_DISTANCE);
     }
 
+    public static Tuple2<Label, TextFieldWithCopyIcon> addCompactTopLabelTextFieldWithCopyIcon(GridPane gridPane,
+                                                                                               int rowIndex,
+                                                                                               int colIndex,
+                                                                                               String title,
+                                                                                               String value,
+                                                                                               boolean withoutBeforeSlash) {
```

Comment:
When used with code completion this naming `withoutBeforeSlash` is not very intuitive. Maybe rename it to `onlyCopyTextAfterDelimiter`